src/api/app/views/layouts/webui2/_breadcrumbs.html.haml

```diff
@@ -1,10 +1,3 @@
 %nav{ "aria-label" => "breadcrumb" }
   %ol.breadcrumb.bg-transparent
-    - @breadcrumbs.each do |breadcrumb|
-      %li.breadcrumb-item{ class: @breadcrumbs.last == breadcrumb && 'active' , "aria-current" => "page" }
-        - if breadcrumb == @breadcrumbs.first
-          %i.fas.fa-home
-        - if breadcrumb[:path]
-          = link_to breadcrumb[:name], breadcrumb[:path]
-        - else
-          = breadcrumb[:name]
+    = render partial: 'breadcrumb_items'
```

Comment:
nitpick: why dont you move this partial on here? doesn't make much sense to have them seperated IMHO